src/components/forms/OfficeHoursForm.tsx

```diff
@@ -284,146 +278,237 @@ export const OfficeHoursForm: FC = () => {
             </Fade>
           </Box>
 
-          <FormControl id='project-name-control' mb={8}>
-            <FormLabel htmlFor='projectName' mb={1}>
-              <PageText fontSize='input'>Project name</PageText>
-            </FormLabel>
-
-            <PageText as='small' fontSize='helpText' color='brand.helpText'>
-              If you&apos;re not asking for help on a project, or simply don&apos;t have a title
-              yet, you can write &quot;N/A&quot;.
-            </PageText>
 
-            <Input
-              id='projectName'
-              type='text'
-              bg='white'
-              borderRadius={0}
-              borderColor='brand.border'
-              h='56px'
-              color='brand.paragraph'
-              fontSize='input'
-              mt={3}
-              {...register('projectName', {
-                maxLength: 255
-              })}
-            />
+          <Controller
+            name='officeHoursRequest'
+            control={control}
+            rules={{ required: true }}
+            defaultValue='Advice'
+            render={({ field: { onChange, value } }) => (
+              <FormControl id='office-hours-request-control' isRequired mb={value === PROJECT_FEEDBACK ? 4 : 8}>
+                <FormLabel htmlFor='officeHoursRequest' mb={2}>
+                  <PageText display='inline' fontSize='input' mb={1}>
+                    Office Hours Request
+                  </PageText>
+                </FormLabel>
 
-            {errors?.projectName?.type === 'maxLength' && (
-              <Box mt={1}>
-                <PageText as='small' fontSize='helpText' color='red.500'>
-                  Project name cannot exceed 255 characters.
+                <PageText as='small' fontSize='helpText' color='brand.helpText'>
```

Comment:
Is this help text really needed or could we remove it?

Reply:
Can check with ESP team on this one... it was added to the spreadsheet as "Choose from the picklist" for helper text, and I had just made a slight adjustment since "picklist" felt confusing. Agree we could potentially remove this, but for sake of not holding this up more, can address separately.